shell/agents/Microsoft.Azure.Agent/Telemetry/MetricHelper.cs

```diff
@@ -0,0 +1,162 @@
+﻿using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.ApplicationInsights.WorkerService;
+using Microsoft.Extensions.DependencyInjection;
+using System.Text.Json;
+
+namespace Microsoft.Azure.Agent;
+
+public class MetricHelper
```

Comment:
We will need to send telemetry in different types, comparing with passing a telemetry instance around, it's better to have a singleton instance that can be used within different types. Serilog is an example:
```
using Serilog;

class Program
{
    static void Main(string[] args)
    {
        // Configure the logger
        Log.Logger = new LoggerConfiguration()
            .WriteTo.Console()
            .CreateLogger();

        // Use the logger
        Log.Information("Hello, Serilog!");
        Log.Warning("This is a warning message.");
        Log.Error("An error occurred.");

        // Ensure to flush and close the log
        Log.CloseAndFlush();
    }
}
```